node/rpcstack.go

```diff
@@ -402,6 +402,10 @@ func (h *virtualHostHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.next.ServeHTTP(w, r)
 		return
 	}
+	if _, exist := h.vhosts["any"]; exist {
```

Comment:
let's add `to_lower`

Reply:
It already does to lower when the value is put into the map. on line 376:

for _, allowedHost := range vhosts {
		vhostMap[strings.ToLower(allowedHost)] = struct{}{}
	}